lib/server.js

```diff
@@ -12,6 +12,11 @@ module.exports = () => {
   const events = new EventEmitter()
   const app = express()
 
+  if (process.env.FORCE_HTTPS) {
+    app.use(require('helmet')())
+    app.use(require('express-sslify').HTTPS({ trustProtoHeader: true }))
+  }
+
```

Comment:
This should be easy to add tests for, but this whole server needs to be wrapped in a function that gets called at runtime so we can change this env var and witness the affect.